VAMobile/src/components/WaygateWrapper.tsx

```diff
@@ -24,7 +25,10 @@ export const WaygateWrapper: FC<WaygateWrapperProps> = ({ children, waygateName
     return (
       <Box mb={theme.dimensions.condensedMarginBetween}>
         <AlertBox border={waygate.type === 'DenyContent' ? 'error' : 'warning'} title={waygate.errorMsgTitle} text={waygate.errorMsgBody}>
-          <ClickToCallPhoneNumber displayedText={displayedTextPhoneNumber(t('8006982411'))} phone={t('8006982411')} a11yLabel={a11yLabelID(t('8006982411'))} />
+          <Box my={theme.dimensions.standardMarginBetween}>
+            <ClickToCallPhoneNumber displayedText={displayedTextPhoneNumber(t('8006982411'))} phone={t('8006982411')} a11yLabel={a11yLabelID(t('8006982411'))} />
+          </Box>
+          {waygate.appUpdateButton === true && <VAButton onPress={openAppStore} label={t('encourageUpdate.update')} buttonType={ButtonTypesConstants.buttonPrimary} />}
```

Comment:
Nitpicky, but isn't this the same as:
```suggestion
          {waygate.appUpdateButton && <VAButton onPress={openAppStore} label={t('encourageUpdate.update')} buttonType={ButtonTypesConstants.buttonPrimary} />}
```